src/main/scala/com/databricks/spark/redshift/RedshiftJDBCWrapper.scala

```diff
@@ -197,7 +197,7 @@ private[redshift] class JDBCWrapper {
    *                                discover the appropriate driver class.
    * @param url the JDBC url to connect to.
    */
-  def getConnector(userProvidedDriverClass: Option[String], url: String): Connection = {
+  def getConnector(userProvidedDriverClass: Option[String], url: String, credentials: Option[(String, String)] = None): Connection = {
```

Comment:
Two nits here:

1. providing a default value here is going to encourage errors. If we don't have a default value, then the compiler will catch any places that we might have forgotten to update the code.
2. since this line is now long, would you mind wrapping the parameters according to our style guide? See https://github.com/databricks/scala-style-guide#spacing-and-indentation